test/hydrated_storage_test.dart

```diff
@@ -163,22 +163,24 @@ void main() {
           hydratedStorage = await HydratedBlocStorage.getInstance(
             storageDirectory: directory,
           );
-          final tasks = Iterable.generate(120, (i) => i).map((i) async {
+          await Stream.fromIterable(
+            Iterable.generate(120, (i) => i),
+          ).asyncMap((i) async {
             final record = Iterable.generate(
               i,
               (i) => Iterable.generate(i, (j) => 'Point($i,$j);').toList(),
             ).toList();
+
             hydratedStorage.write(token, record); // no await here
 
             hydratedStorage = await HydratedBlocStorage.getInstance(
               storageDirectory: directory,
-            ); // basically refreshes cache
+            ); // basically does nothing now
```

Comment:
if it does nothing can we remove it haha?

Reply:
idk, it can be useful if we introduce a way to create custom boxes: some for encrypted bloc's, some for overall separation of concerns

Reply:
alright fair enough 👍